development.cfg

```diff
@@ -2,3 +2,9 @@
 extends =
     test-plone-4.3.x.cfg
     https://raw.github.com/4teamwork/ftw-buildouts/master/plone-development.cfg
+
+find-links +=
+    http://psc.4teamwork.ch/simple
```

Comment:
Do we have private dependencies?
I'd prefer extending https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/4teamwork-psc.cfg in the `sources.cfg`, which already includes the `find-links` extension.